### ISSUE TITLE
Prepare your Node for Time Series Index (TSI) Conversation 

### ISSUE THREAD

**Issue by DarrenMendalka** [influxdata/docs.influxdata.com#2201]
When customers are converting to TSI, here are steps that are needed to remove old data and indexing in addition to the ones covered in our documentation.  The following steps below need to be added to the document as clients have had to redo the upgrade because these steps are not added in the upgrade documentation. 

This is the document that I found and speaks about Conversion in step 5.

https://docs.influxdata.com/enterprise_influxdb/v1.7/administration/upgrading/

1.7.4 and beyond. 

Below are the steps that need to be added as part of this. 

Steps to be included for TSI conversion. 

> Stop the influx process on data node.
> Remove all _series directories: These are located at /data/<dbName>/_series
> Remove all index directories: These are located at /data/<dbName/<rpName>/<shardID>/index
> Rebuild the TSI index: influx_inspect buildtsi -datadir <data_dir> -waldir <wal_dir>
> Start the influx process on data node.
> Allow HH queue to drain from the other data nodes
> Move to next data node and repeat above steps once HH is drained and back online.

NOTE : Very important that the data node comes back on-line before and completely drained the HH queue before starting the next data node.  This is to ensure this is a rolling process

**Comment by stevebang** [comment]
@gunnaraasen @DarrenMendalka This is for conversion from "in-mem" to "tsi", not from TSI Preview right? We'll need to make sure these two paths are clearly identified. [Anyone still on TSI Preview? Can we get rid of that path?]

**Comment by gunnaraasen** [comment]
Yeah, this should be for converting in-mem to tsi using the current version.

When we cut 1.8 release, we could reduce the TSI-preview section to a snippet that points people to the full section TSI-preview upgrade section in the 1.7 docs.

**Comment by DarrenMendalka** [comment]
@kelseiv  Can we close this one?
